dbt/include/athena/macros/materializations/models/table/table.sql

```diff
@@ -50,7 +50,7 @@
     {{ rename_relation(old_relation, old_relation_bkp) }}
     {{ rename_relation(tmp_relation, target_relation) }}
 
-    {{ drop_relation(old_relation_bkp) }}
```

Comment:
Why did you change this one ?

Reply:
@chrischin478 did you see this one? any specific reason why 
`{{ drop_relation(old_relation_bkp) }}` was replaced with `adapter.delete_from_glue_catalog(old_relation_bkp)`?

In this case if the old relation bkp exist we want to drop it and prune the old data too, therefore drop_relation seems more appropriate here. Could you please change that?

Reply:
@nicor88 I've just pushed the `drop_relation` change